src/main/java/duke/Main.java

```diff
@@ -0,0 +1,33 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.controller.MainWindow;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+
+/**
+ * Main entry point to run the java app
+ */
+public class Main extends Application {
+    private static final String FILE_PATH = "data/tasks.txt";
```

Comment:
Noticed this is different from your file path in the Duke class, maybe can try and standardize it!